IEG/IEG-IOC-02App/Db/Makefile

```diff
@@ -0,0 +1,18 @@
+TOP=../..
```

Comment:
Only one of this device physically exists - probably doesn't make sense to have multiple IOCs.